.github/workflows/ci.yml

```diff
@@ -68,7 +64,7 @@ jobs:
       - name: Test with pytest
         run: |
           ulimit -c unlimited  # Enable core dumps to be captured
-          cp /usr/lib/python3/dist-packages/libtorrent*.so $GITHUB_WORKSPACE/deluge
+          cp $LD_LIBRARY_PATH/python3*/site-packages/libtorrent*.so $GITHUB_WORKSPACE/deluge
```

Comment:
this lib copy should not be needed if using pip install

Reply:
Removed.